src/app/modules/entry-live/entry-live.module.ts

```diff
@@ -82,6 +83,9 @@ import { StatusBulletComponent } from './views/live-discovery-table/users-table/
     InputHelperModule,
     SelectButtonModule,
     AutoCompleteModule,
+    PopupWidgetModule,
```

Comment:
@diamond-darrell - we can probably remove the PopupWidgetModule

Reply:
fixed